### DISCUSSION TITLE
[Alpha] ProgressBar

### DISCUSSION THREAD

**Discussion by sjur-gr** [navikt/Designsystemet#2582] [Komponenter]
## Velkommen til vurdering av ProgressBar 🙌
`ProgressBar` brukes til å fortelle hvor langt en bruker har kommet, eller hvor langt et system har kommet, i "noe".

Forslaget her er å tilby en enkel måte å kommunisere fremdrift på. Med tekst og/eller en indikator.
<img width="958" alt="Skjermbilde 2023-12-22 kl  08 45 23" src="https://github.com/navikt/aksel/assets/82579704/9893e0a9-5b40-4d4d-9b36-4d84f08366f8">

Vi har lagt et eksempel på komponenten [ProgressBar i Alpha UI kit](https://www.figma.com/file/Jw3DCrpJgK0uiG7v92Wdnk/Alpha-UI-kit?type=design&node-id=209-31259&mode=design) som dere kan teste ut.

For å finne ut om vi skal inkludere `ProgressBar` som en komponent i Aksel trenger vi deres hjelp. Vi er på jakt etter bevis på behov i løsninger som er i prod/demo. Vi ønsker skjermbilder (uten sensitive opplysninger) og en forklaring av konteksten.

Vi holder ikke aktivt øye med denne diskusjonen, så gi oss et hint om du legger til noe her. Når vi har fått bevis på minst 3 tilfeller av bruk vil vi gå videre med komponenten. Del dette gjerne med team du tror `ProgressBar` vil passe for.

**Comment by BeoPeter** [comment]
![image](https://github.com/navikt/aksel/assets/99338525/39258b1d-fd2a-4c82-9eb5-7894836c921e)

Som en av de siste hypotesene vi hadde i team IA, så satte vi ut en variant av progress bar på en seksjon av innholdet på forebygge fravær. Målet var å gi brukere en indikasjon på hva som var påbegynt inne på sidene, hvor langt de var på vei, og gjøre det lett for arbeidsgivere som var sjeldent inne der, å raskt plukke opp tråden der de slapp sist. 
Målingen vi hadde satt opp viste at brukere interagerte med dette innholdet, og at antallet som valgte å komme tilbake og fortsette der de slapp, økte. Metrikkene eksisterer ikke lenger, da IA-metrikk siden er lagt ned, men det går ann å finne tilbake til de i datafortelling.
Siden man finner disse er under min-side arbeidsgiver - forebygge fravær

**Comment by sjur-gr** [comment]
[Komponenten ligger i Aksel nå](https://aksel.nav.no/komponenter/core/progressbar)
